### ISSUE TITLE
Not creating sass file

### ISSUE THREAD

**Issue by joshlasdin** [snaerth/create-react-component-folder#6]
I'm having an issue when running with the `-s` or `--sass` param that it's not creating a sass file for my component. The output in the terminal says that it does (pasted below), but no such file exists in the folder. If I remove the flag, it does create a css file as expected.

```
/path/to/my/components/shape
  └─ index.js
  └─ shape.js
  └─ shape.test.js
  └─ shape.sass
```

**Comment by snaerth** [comment]
@joshlasdin Strange! Thank you for letting me know. I will get right on it

**Comment by snaerth** [comment]
@joshlasdin.  This has been fixed in the newest version v0.1.15. Thank you again for letting me know and enjoy 👍

**Comment by joshlasdin** [comment]
Awesome, thanks!
